web/src/tippyjs.js

```diff
@@ -78,6 +112,15 @@ const LONG_HOVER_DELAY = [750, 20];
 // distracting users unnecessarily.
 const EXTRA_LONG_HOVER_DELAY = [1500, 20];
 
+function message_list_tooltip(target, props) {
+    delegate("body", {
+        target,
+        appendTo: () => document.body, // Since the bug is only present in these tooltips
+        plugins: [store_message_list_instances_plugin], // Stores and track all the instances created.
```

Comment:
We try not to use use comments in the same line as code. Let's just add this comment and remove the comments here.

```
        // Tooltips present outside of message list table in DOM don't get effected by `rerender` but
        // since their original reference is removed, their position is miscalculated and they get
        // placed at top left of the window. To avoid this, we use this wrapping function.
```